test/scripts/filters/injector.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+describe('Injector', () => {
+  const Hexo = require('../../../lib/hexo');
+  let hexo, injectorFilter;
+
+  beforeEach(() => {
+    hexo = new Hexo();
+    injectorFilter = require('../../../lib/plugins/filter/after_render/injector').bind(hexo);
```

Comment:
I have no idea about how to invalidate the cache through each test case. And it is why the travis ci build failed...